src/ci/utils.py

```diff
@@ -234,7 +236,7 @@ def runValgrind(moduleName):
     printHeader(moduleName, 'valgrind')
     tests = []
     reg = re.compile(
-        ".*unit_test|.*unit_test.exe|.*integration_test|.*integration_test.exe")
+        ".*unit_test|.*unit_test.exe|.*integration_test|.*interface_test|.*integration_test.exe|.*interface_test.exe")
```

Comment:
syscheck interface tests have address sanitizer not working with valgrind together

Reply:
for address sanitizer we will use the test tool.